react/features/base/conference/middleware.js

```diff
@@ -235,6 +245,52 @@ function _connectionFailed({ dispatch, getState }, next, action) {
     return result;
 }
 
+/**
+ * Returns whether or not a CONNECTION_FAILED action is for a possible split
+ * brain error. A split brain error occurs when at least two users join a
+ * conference on different bridges. It is assumed the split brain scenario
+ * occurs very early on in the call.
+ *
+ * @param {Function} getState - The redux function for fetching the current
+ * state.
+ * @param {Action} action - The redux action {@code CONNECTION_FAILED} which is
+ * being dispatched in the specified {@code store}.
+ * @private
+ * @returns {boolean}
+ */
+function _isMaybeSplitBrainError(getState, action) {
+    const { error } = action;
+    const isItemNotFoundError = error
+        && error.message === 'item-not-found'
+        && error.details
+        && error.details.shard_changed;
+
+    if (isItemNotFoundError) {
```

Comment:
Totally not a big deal, but the name is not 100% accurate :P I'd name it isShardChangedError. item-not-found is more general than that